### DISCUSSION TITLE
GitHub Sponsorship

### DISCUSSION THREAD

**Discussion by wjkoh** [gotenberg/gotenberg#914] [Q&A] [answered]
I regularly sponsor open source projects with small monthly donations via GitHub sponsorship. Will this sponsorship feature be available for Gotenberg? I can't find the sponsor button at the moment. Thanks!

**Comment by gulien** [comment, marked as answer]
Hello @wjkoh,

Thanks for your interest! I should setup a sponsor page, you're not the first one to ask 😄

**Reply by gulien** [reply]
@wjkoh finally found the time: https://github.com/sponsors/gulien! Thanks for considering sponsoring Gotenberg 😄
